### PR TITLE
[JA] Feedback Report translations

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Post-Edited Machine Translation by Alconost
[See review request for report-2022-q4.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/58360)
[See review request for report-2022-q3.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/58361)

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
@agektmr FYI - the TOC on the left is partly not localised - anything we can do to help with this (how is this generated?)
image

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q3.md [hunk 1]
```diff
@@ -0,0 +1,945 @@
+---
+layout: layouts/doc-post.njk
+title: フィードバック レポート   - 2022 年第 3 四半期
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q3.md:3]:
```suggestion
title: フィードバック レポート - 2022 年第 3 四半期
```

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q4.md [hunk 2]
```diff
@@ -0,0 +1,680 @@
+---
+layout: layouts/doc-post.njk
+title: フィードバック レポート   - 2022 年第 4 四半期
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q4.md:3]:
```suggestion
title: フィードバック レポート - 2022 年第 4 四半期
```

**Review by agektmr — APPROVED** [review]
Just a couple of nits but LGTM.

**Review by anusmitaray — APPROVED** [review]

**Pull request merged**